packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -68,7 +68,7 @@ abstract class KeyBase extends Resource implements IKey {
    * Defines a new alias for the key.
    */
   public addAlias(alias: string): Alias {
-    return new Alias(this, 'Alias', { aliasName: alias, targetKey: this });
+    return new Alias(this, `Alias${alias}`, { aliasName: alias, targetKey: this });
```

Comment:
Not a particularly clean solution.
Maybe adding an `aliasCounter` property to the Key and appending it instead of the name would be more preferable?

Reply:
This is good, because it's stable. Using counters will lead to problems if people change the order of a list of aliases, since all of their logical ids would change.

Reply:
Good point.

Could there be issues with valid `aliasName` values that would result in an invalid logical ID? I should probably slug the name if we're keeping it

Reply:
Don't worry about it, that happens automatically.

Reply:
Alright 👍 

And thanks for the fixes!